Daymorelah/RideMyWay
# #2178134 Only friends should view and request each other's ride.

#### What does this PR do?
Allows a user to view details of rides owned only by a friend of the user
Allows a user to make requests to rides owned only by a friend of the user 
#### Description of Task to be completed?
* Ensure correct API versioning is used
* Only friends can view each other's ride detail and request to ride
#### How should this be manually tested?
* clone the repo and `cd` into the project's directory.
* Run `npm test` from the console and ensure `NODE_ENV` is set to `test`
